internal/audit/csvformatter.go

```diff
@@ -39,21 +39,20 @@ func (f CsvFormatter) Format(entry *Entry) ([]byte, error) {
 	// source of the official loggers.
 	auditedEntries := entry.Data["audited"]
 	invalidEntries := entry.Data["invalid"]
-	packageCount := entry.Data["num_audited"]
-	numVulnerable := entry.Data["num_vulnerable"]
 	buildVersion := entry.Data["version"]
+	vulnerableEntries := entry.Data["vulnerable"]
 
-	if auditedEntries != nil && invalidEntries != nil && packageCount != nil && numVulnerable != nil && buildVersion != nil {
+	if auditedEntries != nil && invalidEntries != nil && vulnerableEntries != nil && buildVersion != nil {
 		auditedEntries := entry.Data["audited"].([]types.Coordinate)
 		invalidEntries := entry.Data["invalid"].([]types.Coordinate)
-		packageCount := entry.Data["num_audited"].(int)
-		numVulnerable := entry.Data["num_vulnerable"].(int)
 		buildVersion := entry.Data["version"].(string)
+		numVulnerable := len(vulnerableEntries.(map[string]interface{}))
+		numPackages := len(auditedEntries)
 
 		var summaryHeader = []string{"Audited Count", "Vulnerable Count", "Build Version"}
 		var invalidHeader = []string{"Count", "Package", "Reason"}
 		var auditedHeader = []string{"Count", "Package", "Is Vulnerable", "Num Vulnerabilities", "Vulnerabilities"}
```

Comment:
Should we also maybe add these "can be updated" packages to the csv output too?? Maybe a column of "Latest non-vulnerable version" or something like that??